src/pages/send-tokens/components/send-max-button.tsx

```diff
@@ -33,10 +35,12 @@ interface SendMaxProps {
 
 export function SendMaxButton(props: SendMaxProps): JSX.Element | null {
   const { isLoadingFee, onClick } = props;
+  const { selectedAsset } = useSelectedAsset();
+  const isStx = selectedAsset?.type === 'stx';
 
-  return !isLoadingFee ? (
-    <SendMaxButtonAction onClick={onClick} />
-  ) : (
+  return isLoadingFee && isStx ? (
     <SendMaxButtonAction onClick={() => toast.error('Unable to calculate max spend')} />
```

Comment:
```suggestion
    <SendMaxButtonAction onClick={() => toast.error('Unable to calculate max. Try again.')} />
```

Message could probably say they should try again.

Reply:
Sounds good.